migrations/20200114-del-column-builds-steps.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}builds`;
+
+module.exports = {
+    // eslint-disable-next-line no-unused-vars
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
```

Comment:
no need for a single transaction

Reply:
I think it is better to be able to roll back if you fail on the way.

Reply:
u can keep it - but no use, as there is only one statement getting executed; it will be useful for multiple statements

Reply:
I see.
I fixed it.